content/docs/5.1/concepts/tikv-in-5-minutes.md

```diff
@@ -30,139 +32,143 @@ The following steps are only for test environments. For production environments
 
 2. Set the TiUP environment variables:
 
-    Redeclare the global environment variables:
+    1. Redeclare the global environment variables:
 
-    ```bash
-    source .bash_profile
-    ```
+        ```bash
+        source .bash_profile
+        ```
 
-    Confirm whether TiUP is installed:
+    2. Confirm whether TiUP is installed:
 
-    ```bash
-    tiup
-    ```
+        ```bash
+        tiup
+        ```
+
+    <!-- Explain the results -->
```

Comment:
@marsishandsome Should we explain what the expected result is?

Reply:
Please delete this line.

Reply:
```suggestion
```